hack/run-locally.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash 
+
+if [ -z "$CLUSTER_DIR" ]; then
+    echo CLUSTER_DIR is required
+    exit 1
+fi
+
+set -o errexit
+set -o nounset
+
+# Install our overrides so the cluster doesn't run the network operator.
+function override() {
+    local cluster_dir=$1
```

Comment:
You use this variable for the first half of the function but then fall back to `$CLUSTER_DIR` in the second half. Probably no need to pass this in as an argument, just use the global everywhere.